test-utils/runtime-tester/src/fuzzing.rs

```diff
@@ -36,7 +36,11 @@ impl Arbitrary<'_> for Scenario {
         let mut scope = Scope::from_seeds(&seeds);
 
         let network_config = NetworkConfig { seeds };
-        let runtime_config = RuntimeConfig { max_total_prepaid_gas: GAS_1 * 100 };
+        let runtime_config = RuntimeConfig {
+            max_total_prepaid_gas: GAS_1 * 100,
+            gas_limit: (GAS_1 as f64 * *u.choose(&[0.01, 0.1, 1., 10., 100.])?) as u64,
+            epoch_length: *u.choose(&[5, 10, 100, 500])? as u64,
```

Comment:
how are these numbers chosen?

Reply:
Intuitively, because I didn't have any ideas on how to do it properly. Open to any suggestions.